frontend/Themes/Store.js

```diff
@@ -76,6 +76,12 @@ class Store {
   setDefaultThemeName(defaultThemeName: ?string) {
     // Don't accept an invalid themeName as a default.
     this.defaultThemeName = getSafeThemeName(defaultThemeName);
+    // Replace theme with new default if user hasn't select theme
+    const storedThemeKey = getFromLocalStorage(LOCAL_STORAGE_THEME_NAME_KEY);
+    if (storedThemeKey !== CUSTOM_THEME_NAME && !Themes.hasOwnProperty(storedThemeKey)) {
+      this.theme = Themes[this.defaultThemeName];
+      this.themeName = this.defaultThemeName;
+    }
```

Comment:
I still have a small concern about the method name (now `setDefaultTheme`) doing a bit too much (eg loading custom theme). I'll just tweak this myself in a follow-up commit though so we don't have to keep going back and forth. I want to respect your time here. 😄 

Hopefully you're okay with this minor adjustment?
```diff
diff --git a/frontend/Themes/Store.js b/frontend/Themes/Store.js
index 27e6bcb..20754d4 100644
--- a/frontend/Themes/Store.js
+++ b/frontend/Themes/Store.js
@@ -27,6 +27,15 @@ class Store {
   themes: { [key: string]: Theme };
 
   constructor(defaultThemeName: ?string) {
+    this.themes = Themes;
+
+    // Load previous custom theme from localStorage.
+    // If there isn't one in local storage, start by cloning the default theme.
+    const customTheme = getFromLocalStorage(LOCAL_STORAGE_CUSTOM_THEME_KEY);
+    if (customTheme) {
+      this.customTheme = deserialize(customTheme);
+    }
+
     this.setDefaultTheme(defaultThemeName);
   }
 
@@ -41,18 +50,10 @@ class Store {
       this.defaultThemeName,
     );
 
-    // Load previous custom theme from localStorage.
-    // If there isn't one in local storage, start by cloning the default theme.
-    const customTheme = getFromLocalStorage(LOCAL_STORAGE_CUSTOM_THEME_KEY);
-    if (customTheme) {
-      this.customTheme = deserialize(customTheme);
-    }
-
     // The user's active theme is either their custom one,
     // Or one of the built-in sets (based on the default).
     this.theme = themeName === CUSTOM_THEME_NAME ? this.customTheme : Themes[themeName];
     this.themeName = themeName;
-    this.themes = Themes;
   }
 
   update(themeName: ?string) {
```